web/source/kmwembedded.ts

```diff
@@ -136,10 +136,7 @@ namespace com.keyman.text {
     opt['attachType'] = 'manual';
     device.app=opt['app'];
     device.touchable=true;
-    device.formFactor='phone';
-    if(navigator && navigator.userAgent && navigator.userAgent.indexOf('iPad') >= 0) device.formFactor='tablet';
-    if(device.app.indexOf('Mobile') >= 0) device.formFactor='phone';
-    if(device.app.indexOf('Tablet') >= 0) device.formFactor='tablet';
```

Comment:
Uh... are we sure this won't affect Android?

https://github.com/keymanapp/keyman/blob/1bdb2d4364a9f571d0c12a8be41f858eafba5745/android/KMEA/app/src/main/assets/android-host.js#L8

https://github.com/keymanapp/keyman/blob/1bdb2d4364a9f571d0c12a8be41f858eafba5745/android/KMEA/app/src/main/assets/android-host.js#L24-L28

https://github.com/keymanapp/keyman/blob/1bdb2d4364a9f571d0c12a8be41f858eafba5745/android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboardJSHandler.java#L25-L29

https://github.com/keymanapp/keyman/blob/1bdb2d4364a9f571d0c12a8be41f858eafba5745/android/KMEA/app/src/main/res/values/strings.xml#L5

https://github.com/keymanapp/keyman/blob/1bdb2d4364a9f571d0c12a8be41f858eafba5745/android/KMEA/app/src/main/res/values-sw600dp/strings.xml#L5

Reply:
Look at line 139 below. Doesn't that do the same thing, just in a much less convoluted manner?

```
 device.formFactor = device.app.indexOf('Tablet') >= 0 ? 'tablet' : 'phone';
```

Reply:
Ah.  I saw the ios-host one, but missed the one earlier in this function.  Got it.